src/isar/storage/uploader.py

```diff
@@ -1,18 +1,66 @@
 import logging
+from dataclasses import dataclass
+from datetime import datetime, timedelta
 from queue import Empty, Queue
 from typing import List
 
+from isar.config import config
 from isar.models.mission_metadata.mission_metadata import MissionMetadata
 from isar.storage.storage_interface import StorageException, StorageInterface
 from robot_interface.models.inspection.inspection import Inspection
 
 
+@dataclass
+class UploaderRetryItem:
+    inspection: Inspection
+    mission_metadata: MissionMetadata
+    storage_handler: StorageInterface
+    __retry_count: int
+    __next_retry_time: datetime = datetime.max
```

Comment:
```suggestion
    _retry_count: int
    _next_retry_time: datetime = datetime.max
```
Internal variables in Python are usually prefixed with one underscore.